network-api/networkapi/wagtailpages/pagemodels/libraries/rcc/detail_page.py

```diff
@@ -64,13 +65,26 @@ def get_context(self, request):
         context["content_type_names"] = self.related_content_types_names
         return context
 
+    def get_preview_template(self, request, mode_name):
+        return "previews/libraries/rcc/detail_page.html"
+
     @cached_property
     def localized_authors(self):
-        rcc_author_profiles = wagtailpages_utils.localize_queryset(
-            profiles.Profile.objects.filter(authored_rcc_articles__detail_page=self)
+        rcc_author_profiles = profiles.Profile.objects.filter(authored_rcc_articles__detail_page=self).order_by(
+            "authored_rcc_articles__sort_order"
         )
+        rcc_author_profiles = wagtailpages_utils.localize_queryset(rcc_author_profiles, preserve_order=True)
```

Comment:
Updated this method to make sure that the profiles are always returned in the CMS sort order